src/Doctrine/Repository/ProjectRepository.php

```diff
@@ -78,11 +79,28 @@ public function load(ProjectId $projectId): Project
         }
     }
 
+    /**
+     * @param ProjectId $projectId
+     * @return bool
+     */
+    public function remove(ProjectId $projectId): bool
+    {
+        $qb = $this->entityManager->createQueryBuilder();
+        $qb->delete();
+        $qb->from(ProjectEntity::class, 'p');
+        $qb->where('p.id = :id');
+        $qb->setParameter('id', $projectId->toString());
+        $qb->getQuery()->execute();
+
+        return true;
```

Comment:
Not a big fan of returning boolean value on success or not. By calling removing we expect this action to do it's job, not maybe do it. If this action would fail this is an exception, so it should be handled with an exception, catched and handled.